base/plugins.cpp

```diff
@@ -717,7 +717,7 @@ QualifiedGameList EngineManager::findGameInLoadedPlugins(const Common::String &g
 	return results;
 }
 
-DetectionResults EngineManager::detectGames(const Common::FSList &fslist) {
+DetectionResults EngineManager::detectGames(const Common::FSList &fslist, int skipADFlags, bool skipIncomplete) {
```

Comment:
A major thing: `g->flags` is `uint32`, thus, your `skipADFlags` must be of this type too.